specification/compute/resource-manager/readme.md

```diff
@@ -316,57 +316,91 @@ These settings apply only when `--python` is specified on the command line.
 
 ``` yaml $(python)
 python:
-  # override the default output folder
-  output-folder: $(output-folder)/python
+  azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
   payload-flattening-threshold: 2
+  package-name: azure-mgmt-compute
+  payload-flattening-threshold: 2
+  clear-output-folder: true
+```
+
+### Python multi-api
+
+Generate all API versions currently shipped for this package
+
+```yaml $(python) && $(multiapi)

```

Comment:
We've chatted about this a little offline, but I'll just reiterate here for the sake of transparency and clarity. :)

I worry about having language specific `multiapi` batches. I'd really rather drive consistency between all of the languages, and push towards having SDKs which have either adopted the multi-api paradigm, or not. For my preference, there would be no imbetween where only some API Versions are supported across each language. :)

So my request here would be to remove the `$(python) &&` portion of the line above.